Bug: oxlint/eslint-plugin-oxlint peer dependency mismatch breaks frontend Docker build

## Problem

`autobot-frontend/package.json` had `oxlint@~1.1.0` alongside `eslint-plugin-oxlint@~1.57.0`. The plugin requires `oxlint@~1.57.0` as a peer dependency, causing `npm ci` to fail with `ERESOLVE` during Docker builds.

## Root Cause

When `eslint-plugin-oxlint` was upgraded to 1.57.0, the `oxlint` package was not upgraded to match.

## Fix Applied

Changed `oxlint` from `~1.1.0` → `~1.57.0` in `autobot-frontend/package.json:105` and regenerated `package-lock.json`.

## Related

- #1809 (Docker containerization)
- #1836 (Full-stack dependency upgrade)

--- a/autobot-frontend/package.json
+++ b/autobot-frontend/package.json
@@ -102,7 +102,7 @@
     "mock-socket": "^9.3.0",
     "msw": "^2.8.8",
     "npm-run-all2": "^8.0.4",
-    "oxlint": "~1.1.0",
+    "oxlint": "~1.57.0",
     "playwright": "^1.54.2",
     "postcss": "^8.5.6",
     "prettier": "3.5.3",

